### DISCUSSION TITLE
Home Assistant - Automation and heating saving - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#54] [Comments]
# Home Assistant - Automation and heating saving - Haade.fr

August returns to the video of March 28 concerning heating savings and home automation works and provides figures on its consumption and savings made since February 2024

https://haade.fr/en/blog/home-automation-and-heating-saving-the-assessment

**Comment by zalwandri78** [comment]
What is the main misconception about installing thermostatic faucets for energy savings? regard <a href=" https://jakarta.telkomuniversity.ac.id/menyelami-dunia-botnet-ancaman-tersembunyi-di-era-digital/ "> Tel U </a>
